### PR TITLE
Updated keystrokes in devtools-guide file

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated keystrokes per issue filed by customer.  
#323

### CODE REVIEW

File: microsoft-edge/devtools-guide.md [hunk 1]
```diff
@@ -75,7 +69,7 @@ These shortcuts control the main DevTools window and should work across all tool
 |:--- |:--- |  
 | Show/Hide DevTools \(opens to last viewed panel\) | `F12`, `Ctrl`+`Shift`+`I` |  
 | Toggle docking \(Undock/Bottom/Right\) | `Ctrl`+`Shift`+`D` |  
-| Open file | `Ctrl`+`P`, `Ctrl`+`O` |
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide.md:78]: This isn't actually correct for the latest version of the EdgeHTML-based browser. See: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide/whats-new#source-file-search-tools. The other changes are fine but I wouldn't make this change. Instead I would let the customer know that if they update Windows to the latest version, this shortcut will work as documented.

**Comment by jm-trd-ms** [comment]
Closed due to peer feedback.

File: microsoft-edge/devtools-guide.md (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/devtools-guide.md]: Updated and added note to clarify Windows validation for shortcuts.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**